### ISSUE TITLE
Missing BUILD file in Apollo 9.0

### ISSUE THREAD

**Issue by YuqiHuai** [ApolloAuto/apollo#15259]
The directory `modules/planning/planning_component/integration_tests` is missing a BUILD file that existed in previous versions, see [build file from Apollo 8.0](https://github.com/ApolloAuto/apollo/blob/v8.0.0/modules/planning/integration_tests/BUILD) and [current directory structure for Apollo 9.0](https://github.com/ApolloAuto/apollo/tree/master/modules/planning/planning_component/integration_tests).

Is there a reason why this BUILD file is omitted?

**Comment by daohu527** [comment]
Yes, the reason is that now the BUILD files are merged. And forgot to add in new BUILD file
